sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -1531,43 +1562,47 @@ public Write to(ValueProvider<String> tableSpec) {
     }
 
     /**
-     * Creates a write transformation from a function that maps windows to table specifications.
-     * Each time a new window is encountered, this function will be called and the resulting table
-     * will be created. Records within that window will be written to the associated table.
+     * Returns a copy of this write transformation, but with the specified table function. The
```

Comment:
While you're at it - clean up these comments? "Returns a copy..." and "Does not modify.." are unnecessary.